packages/server/src/utils/saml.ts

```diff
@@ -0,0 +1,90 @@
+import { ServiceProvider, IdentityProvider, setSchemaValidator } from 'samlify';
+import * as validator from '@authenio/samlify-xmllint-wasm';
+import { readFile } from 'fs-extra';
+import config from '../config';
+import { PostBindingContext } from 'samlify/types/src/entity';
+import { _ } from '@joplin/lib/locale';
+import { SamlRelayState } from './types';
+
+// Checks if SAML support is enabled.
+//
+// Throws an error otherwise.
+function checkIfSamlIsEnabled() {
```

Comment:
Please use arrow functions only (see coding style guide). All functions here should be converted to arrow functions